src/server.ts

```diff
@@ -1,11 +1,12 @@
 import App from '@/app';
 import AuthRoute from '@routes/auth.route';
+import GroupsRoute from './routes/group.route';
 import IndexRoute from '@routes/index.route';
 import UsersRoute from '@routes/users.route';
 import validateEnv from '@utils/validateEnv';
 
 validateEnv();
 
-const app = new App([new IndexRoute(), new UsersRoute(), new AuthRoute()]);
+const app = new App([new IndexRoute(), new UsersRoute(), new AuthRoute(), new GroupsRoute()]);
```

Comment:
GroupsRoute routes collection should be added before AuthRoute since auth middleware should be executed first before any other user/group crud operation